manifests/s/Samsung/DataMigration/14.0.162/Samsung.DataMigration.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 14.0.162

```

```suggestion
PackageVersion: 2.0
```
This must match control panel. The version folder will need to be renamed

---

Please update file name and path to match this change.